### ISSUE TITLE
open-source timeline

### ISSUE THREAD

**Issue by 31b14fee** [KeiChiTse/QPT-V2#2]
May I ask when the inference and training code will be released? I tried loading the model using the HiViT code, but the results were not satisfactory. In addition, I noticed that the quality_regressor module contains fc_cls.0 and fc_cls.3. Aside from the two linear layers and the GeLU mentioned in the paper, there seems to be an extra layer—could this be a dropout layer?
